react-native-app/components/Profile.tsx

```diff
@@ -62,6 +66,14 @@ const Profile: React.FC<Props> = ({
     })
 
   }
+
+  async function loadNextCheatMealPage(prevPage?: UserCheatMealResponse) {
+
+    let page = prevPage || {records:[],pageBreakKey:undefined,pageSize:20}
+    let response = await new CheatMealService().getUserChealMealFeedPage(profileOwner.userId,page)
+    response.records.sort((a,b)=>a.created<b.created?1:-1)
+    return response;
+  }
   
   async function loadNextPage(prevPage?: UserSnitchesResponse) {
```

Comment:
Since we now have two "loadPage" methods can we also specify that this one is specifically for snitches? 
```suggestion
  async function loadNextSnitchPage(prevPage?: UserSnitchesResponse) {
```